src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService .cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using UserId = System.Guid;
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
+{
+    public class ThemeService : IThemeService
+    {
+        private readonly ApplicationDbContext _context;
+
+        public ThemeService(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        public List<Theme> GetThemes(UserId userId)
+        {
+            // TODO This method simply returns all Themes as we currently do not have a concept of how a user
+            // TODO is connected to Themes for the purpose of administration. Once this has been modelled then
+            // TODO this method will need altered reflect this. 
+            var result = _context.Themes.Select(t => new Theme
```

Comment:
Probably need to have a discussion if it's right to be importing the Content's ApplicationDbContext here or if that needs to be turned into something more 'common'. `t` in this instance is a Theme from the Content.Model project

Reply:
This should be fine, eventually the content api wont be driven by that database therefore this will become its primary user.